cmd/feedback/feedback_test.go

```diff
@@ -66,7 +68,9 @@ func TestFeedbackCommand(t *testing.T) {
 
 		clients := shared.NewClientFactory(clientsMock.MockClientFactory())
 
+		_surveys := SurveyStore
 		SurveyStore = surveys
+		defer func() { SurveyStore = _surveys }()
```

Comment:
note: After adding some Command Table Tests, I noticed that the tests started failing when ran together. The reason is that the current tests clobber the global `SurveyStore`. This code backs up and restores the original `SurveyStore` after each test. 

🧪 In a follow-up PR, I'd like to refactor these tests to all use the Command Table Test.